pyproject.toml

```diff
@@ -37,6 +37,9 @@ classifiers = [
 dynamic = ["version", "readme"]
 
 [project.optional-dependencies]
+all = [
```

Comment:
Do we need this package at all? I removed it in https://github.com/google/flax/pull/4885.